src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASE_PRICE = PRICE_PER_DAY * days;
```

Comment:
The variable `BASE_PRICE` is not a constant value, because it's calculated, so use camel case for naming.

Reply:
and the name base price doesn't fully say what's stored there, because as a developer I'm asking questions: "Price for what? Price for someday?". The variable stores the final price without discount, maybe it would be better to call it that.